src/options.cpp

```diff
@@ -2808,7 +2821,9 @@ std::string options_manager::show( bool ingame, const bool world_options_only,
             }
         }
 
-        calcStartPos( iStartPos, iCurrentLine, iContentHeight, page_items.size() );
+        if( recalc_startpos ) {
+            calcStartPos( iStartPos, iCurrentLine, iContentHeight, page_items.size() );
```

Comment:
The flag is just to prevent the list from scrolling when mousing over options (when there are lots of options, mousing over anywhere but the center would cause a scroll, which puts the selection on a different option). I will add this to the resize callback, though.